make.sh

```diff
@@ -112,6 +127,14 @@ sed -i.bak "$(($line_number + 1))i\\"$'\n'"\
   pod 'shared', :path => '../shared'\\
 " podfile
 
+# Remove unnecessary files after generating the iOS module
+rm -rf .github
+rm -f README.md
+rm -f bitrise.yml
+rm -f codemagic.yaml
+rm *.bak
+rm *.sh
```

Comment:
@phongvhd93 still some files to remove:
- https://github.com/nimblehq/kmm-templates/blob/feature/21-write-script-for-removing-unnecessary-files-and-folders/sample/ios/LICENSE